extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -270,14 +270,34 @@ export class ConsentUI {
    */
   createPromptIframeFromSrc_(promptUISrc) {
     const iframe = this.parent_.ownerDocument.createElement('iframe');
+    let sandbox = 'allow-scripts';
     iframe.src = assertHttpsUrl(promptUISrc, this.parent_);
-    iframe.setAttribute('sandbox', 'allow-scripts');
+    const allowSameOrigin = this.allowSameOrigin_(
+        iframe.src, window.location.href);
```

Comment:
Please don't use `window.location.href` directly. I know you referred to the amp-iframe usage, but we should have changed the usage in amp-iframe as well : )